int8 cascade_type
int32 distance_mm
float64 horizontal_ang
float64 vertical_ang
time time
